py/objint.c

```diff
@@ -388,67 +388,93 @@ mp_obj_t mp_obj_int_binary_op_extra_cases(mp_binary_op_t op, mp_obj_t lhs_in, mp
 }
 
 // this is a classmethod
-STATIC mp_obj_t int_from_bytes(size_t n_args, const mp_obj_t *args) {
-    // TODO: Support signed param (assumes signed=False at the moment)
-    (void)n_args;
+STATIC mp_obj_t int_from_bytes(size_t n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
+    enum { ARG_signed };
```

Comment:
Not 100% sure, but seeing this code is repeated below, it might save code size to put it in a small function which does the parsing and the check.